ustriage/ustriage.py

```diff
@@ -302,9 +302,45 @@ def handle_webbrowser(open_in_browser, url):
         time.sleep(5)
 
 
+def print_bug_line(text, task, postponed_bugs):
+    """Format each bug line, like strikethrough for postponed bugs."""
+    if task.number in postponed_bugs:
+        text = '\u0336'.join(text) + '\u0336'
```

Comment:
nitpick: `'\u0336'` could be in a constant to avoid repetition of the magic number in the future if we need to use it somewhere else.

Reply:
yep, great suggestion ... done